scripts/provisioning/Vagrantfile

```diff
@@ -598,6 +598,43 @@ def setup(node, cpu_nr, mem_mb, disk_nr = 0, disk_size = 1)
     end
   end
 
+  # VMware Desktop --------------------------
```

Comment:
What is the change for?
I don't see the relevance of this change and the title of this PR.

Reply:
@huanghua78 , I run Linux in VMware VMs on Windows PC under Vagrant. On Windows, the shared folder (where all the sources are stored) is mounted by SMB/Samba/CIFS protocol. (NFS is not supported by Vagrant.) This patch fixes Motr build when the sources are mounted via CIFS.

Hope this helps? If not, don't hesitate to ask more. For more information about provisioning VMs for Motr with Vagrant refer to https://github.com/Seagate/cortx-motr/tree/main/scripts/provisioning.

Reply:
I have updated description here also so that it would be more clear what this patch is for. I agree - it was not quite clear before.

Reply:
Ah, sorry - looks like you are asking about another patch. Yes, it had his own PR - https://github.com/Seagate/cortx-motr/pull/268. But I closed it as it seemed more convenient to have all these patches in one PR as they all are related.

Reply:
OK.

I used Vagrant in the past, but not now. I can not verify this patch myself.

Reply:
No worries - I have verified it. :)

And @chumakd (the author of our Vagrantfile) has approved it also. We continue to work with @chumakd on further improvements, so there might be another patch(es) in future...